client/package.json

```diff
@@ -17,6 +17,7 @@
     "test": "tape -r esm test/**/*.test.js | tap-spec"
   },
   "dependencies": {
+    "@types/node": "^13.7.7",
```

Comment:
Why is this needed?

Reply:
I was getting an error when referencing `process.env` saying that process wasn't defined, so this is the fix.

Reply:
Actually I can't reproduce that error, it seems to be working if I remove that dependency (and do a fresh install). So I can remove this.

Reply:
Looking at how Vue accomplishes this injection (and using `vue inspect` to look at the DefinePlugin section), it does seem like you shouldn't need to bring in any external support for this. Essentially, Webpack is injecting a `process.env` variable into the global scope, and so it should "just work" (though, I'm not entirely getting how TypeScript is ok with this, without some kind of `declare` statement :thinking:).